components/cdc/src/delegate.rs

```diff
@@ -522,29 +524,24 @@ impl Delegate {
         requests: Vec<Request>,
         old_value_cb: &OldValueCallback,
         old_value_cache: &mut OldValueCache,
+        statistics: &mut Statistics,
         is_one_pc: bool,
     ) -> Result<()> {
         let txn_extra_op = self.txn_extra_op;
-        let mut read_old_value = |row: &mut EventRow, read_old_ts| {
+        let mut read_old_value = |row: &mut EventRow, read_old_ts| -> Result<()> {
             if txn_extra_op == TxnExtraOp::ReadOldValue {
                 let key = Key::from_raw(&row.key).append_ts(row.start_ts.into());
-                let start = Instant::now();
-                let (old_value, statistics) = old_value_cb(key, read_old_ts, old_value_cache);
+                let old_value = old_value_cb(key, read_old_ts, old_value_cache, statistics)?;
```

Comment:
What will happen if it returns an error?

Reply:
The delegate will be deregistered.

> When will `old_value_cb` return an error?
Currently errors only come from RocksDB APIs.